doc/microsite/src/main/tut/distage/distage-framework-docker.md

```diff
@@ -19,3 +20,213 @@ libraryDependencies += "io.7mind.izumi" %% "distage-framework-docker" % "$izumi.
 ```
 
 @@@
+
+
+### Overview
+
+Use of `distage-framework-docker` generally follows:
+
+- creating `ContainerDef`s for the containers the application requires
+- declaring a `ModuleDef` that declares the container component
+- including the `DockerSupportModule` in the application's modules
+
+First, the required imports:
+
+```scala mdoc
```

Comment:
Generally use `mdoc:to-string` due to `toString` method having a better output for most types in this project compared to [pPrint](http://www.lihaoyi.com/PPrint/) used in default mdoc mode